### ISSUE TITLE
fancontrol.sys being blocked by antivirus

### ISSUE THREAD

**Issue by CyberReaper93** [Rem0o/FanControl.Releases#1555]
I am currently running the most up to date version, V146, and none of the updates seem to help this issue. Once the application launches, my antivirus picks it up and immediately prevents fancontrol.sys from running which in turn of course means none of the settings work. Not sure what caused this, didn't happen in previous iterations, but I thought I should bring it up for a hopeful fix besides changing or suspending certain aspects of my AV. I tried to make an exception, however, that will not work given how this AV functions.

[log.txt](https://github.com/Rem0o/FanControl.Releases/files/10775181/log.txt)

![Capture](https://user-images.githubusercontent.com/73723232/219899348-feb3cb5e-49e7-4135-8935-355447d4e834.PNG)

![Capture1](https://user-images.githubusercontent.com/73723232/219899350-47c18b5e-6cb1-4f1f-a6a2-a676b2139500.PNG)

Specs:
AMD Ryzen 5 5600X
Nividia RTX 2060
16GB DDR4 3200

**Comment by Rem0o** [comment]
duplicate of https://github.com/Rem0o/FanControl.Releases/issues/1555
